client/src/main/java/org/eclipse/hono/client/CommandTargetMapper.java

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.client;
+
+import org.eclipse.hono.client.impl.CommandTargetMapperImpl;
+import org.eclipse.hono.util.DeviceConnectionConstants;
+
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
+import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * A component for mapping an incoming command to the gateway (if applicable)
```

Comment:
I think "if applicable" needs a bit more explanation here (and in the following javacdocs of this interface). It think it is an integral aspect, and it would help, explaining a bit more when it might be applicable.

Reply:
I've extended the javadoc here.

Reply:
did you?

Reply:
Sorry, I mistook this to be aimed at the javadoc of the `getTargetGatewayAndAdapterInstance` method. I'll update both javadocs (class and method)...

Reply:
done